src/main/java/com/googleintern/wfm/ruleengine/action/GeneralRulesForWorkgroupId.java

```diff
@@ -0,0 +1,76 @@
+package src.main.java.com.googleintern.wfm.ruleengine.action;
+
+import com.google.common.collect.*;
+import src.main.java.com.googleintern.wfm.ruleengine.model.FilterModel;
+import src.main.java.com.googleintern.wfm.ruleengine.model.PoolAssignmentModel;
+import src.main.java.com.googleintern.wfm.ruleengine.model.RuleModel;
+import src.main.java.com.googleintern.wfm.ruleengine.model.UserPoolAssignmentModel;
+
+import java.util.List;
+import java.util.Set;
+
+/**
+ * GeneralRulesForWorkgroupId class is used to create rules that can apply to all users from the
+ * same work group.
+ */
+public class GeneralRulesForWorkgroupId {
```

Comment:
Would rename this to clarify what it does.
Possible names: WorkgroupIdRuleGenerator, GeneralRuleGenerator, BasicRuleGenerator,...